### PR TITLE
build: update deps

### PR DESCRIPTION
**Pull request by juanpicado**
**Type:** build

**Description:**

Update dependencies
 - Fix broken test due this upgrade
 - Update required config
 - Fix sec vulnerabilities

### CODE REVIEW

**Comment by priscilawebdev** [comment]
I fixed the types issues but I still have to check some stuff... it might be a good idea to add visual tests btw 😜

**Comment by priscilawebdev** [comment]
Still have to fix e2e tests

**Comment by juanpicado** [comment]
> Still have to fix e2e tests

great !! 🚀 ..

**Comment by priscilawebdev** [comment]
@juanpicado I managed to fix the e2e tests, but I noticed some strange problems that I will check later

File: test/e2e/e2e.spec.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ describe('/ (Verdaccio Page)', () => {
   // this might be increased based on the delays included in all test
   jest.setTimeout(20000);
 
-  const clickElement = async function(selector, options = { button: 'middle', delay: 100 }) {
```
Comment (priscilawebdev) [inline test/e2e/e2e.spec.js:9]: no idea the reason, but this middle is causing some issues

File: src/utils/package.ts [hunk 2]
```diff
@@ -94,6 +94,14 @@ export function getRecentReleases(time: Time = {}): Time[] {
   return recent.slice(recent.length - 3, recent.length).reverse();
 }
 
-export function getAuthorName(authorName: string): string {
-  return authorName.toLowerCase() === 'anonymous' ? i18next.t('author-anonymous') : authorName;
+export function getAuthorName(authorName?: string): string {
+  if (!authorName) {
+    return i18next.t('author-unknown');
```
Comment (priscilawebdev) [inline src/utils/package.ts:99]: this should not be part of this PR scope, but I ended up adding it ..I'm sorry for that

File: src/pages/home/PackageList/Package/styles.ts [hunk 3]
```diff
@@ -110,7 +110,7 @@ export const PackageListItemText = styled(ListItemText)({
   paddingRight: 0,
 });
 
-export const Description = styled(Typography)<{ theme?: Theme }>(({ theme }) => ({
+export const Description = styled(MuiText)<{ theme?: Theme }>(({ theme }) => ({
```
Comment (priscilawebdev) [inline src/pages/home/PackageList/Package/styles.ts:113]: I have to check this MuiText later on... for sure I have to omit the heading types.

**Review by priscilawebdev — DISMISSED** [review]

File: test/e2e/e2e.spec.js (hunk 1 above)
Comment (juanpicado) [reply test/e2e/e2e.spec.js:9]: Mmmmm ... ok, let me dive a bit more into it, the thing is works locally 😢 .... thanks for the hint @priscilawebdev 👍

**Review by priscilawebdev — APPROVED** [review]
